src/js/utils/dom.js

```diff
@@ -36,7 +36,7 @@ function isNonBlankString(str) {
  *         Throws an error if there is whitespace in the string.
  */
 function throwIfWhitespace(str) {
-  if ((/\s/).test(str)) {
+  if (str.indexOf(' ') >= 0) {
```

Comment:
not using regex is faster.

Reply:
A comment explaining usage would helpful here as well.